app/schema/widgets/relationship_widget.py

```diff
@@ -0,0 +1,58 @@
+from app.globals import get_answer_store
+from app.libs.utils import ObjectFromDict
+from app.schema.widgets.multiple_choice_widget import MultipleChoiceWidget
+
+from flask import render_template
+
+from flask_login import current_user
+
+
+class RelationshipWidget(MultipleChoiceWidget):
+    def __init__(self, name):
+        super().__init__(name)
+        self.type = 'relationship'
+
+    def render(self, answer_state):
+        widget_params = self.build_widget_params(answer_state)
+        return render_template('/partials/widgets/relationship_widget.html', **widget_params)
+
+    def build_widget_params(self, answer_state):
+        answer_instance = answer_state.group_instance + answer_state.answer_instance
+        current_person = get_answer_store(current_user).filter(answer_id='household-full-name',
```

Comment:
Is there any way the answer_store can be passed to the widgets, rather than pulling them in? It would save mocking out everything in tests.